Makefile.am

```diff
@@ -61,6 +65,8 @@ if FATAL_WARNINGS
   COMMON_CXXFLAGS += -Werror
   COMMON_PROTOBUF_CXXFLAGS += -Werror -Wno-error=unused-parameter
   COMMON_TESTING_FLAGS += -Werror
+  COMMON_TESTING_PROTOBUF_FLAGS = $(COMMON_TESTING_FLAGS) \
```

Comment:
Please can you move the "COMMON_TESTING_PROTOBUF_FLAGS = $(COMMON_TESTING_FLAGS)" bit outside the if, like COMMON_TESTING_FLAGS is done. Then just add the -Werror -Wno-error=unused-parameter inside the if; just like COMMON_PROTOBUF_CXXFLAGS. I think you'll find this currently fails to compile if you do --disable-fatal-warnings.